### DISCUSSION TITLE
[Help]: Carousel focus event not behaving as expected

### DISCUSSION THREAD

**Discussion by hunterbecton** [davidjerleke/embla-carousel#1071] [Help] [answered]
### Summary

When I tab to focusable elements within each slide, the carousel doesn't behave as I expect. From what I understand from the docs, the default callback fires the slideFocus event and scrolls to the focused element. So does this mean Embla should automatically move to the slide with the focused element? If not, how do I began to implement this behavior?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [X] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://stackblitz.com/edit/stackblitz-starters-2xhj2u?file=app%2F_components%2FSlideshow.tsx

**Comment by davidjerleke** [comment, marked as answer]
@hunterbecton thanks for your question. I found the following problem in your code: You've missed to set `overflow: hidden;` on your viewport element. Maybe that's why?

```tsx
<div className="h-full" ref={emblaRef}> {/* <-- Add overflow: hidden; here */}
```

**Reply by hunterbecton** [reply]
That's it! It never crossed my mind that it was a CSS issue. Thank you!
